### PR TITLE
Note that ACR is not currently supported

### PR DESCRIPTION
**Pull request by landongrindheim**
### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by landongrindheim** [comment]
Verified in preview environment: 

image

This renders the way I'd expect.

**Comment by cmwilson21** [comment]
@landongrindheim Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md [hunk 1]
```diff
@@ -776,6 +776,12 @@ registries:
 
 ### `docker-registry`
 
+{% note %}
+
+**Note:** Azure Container Registry is currently not supported
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md:781]: Using the active voice and adding a full stop to the end of the note.
```suggestion
**Note:** We currently don't support the Azure Container Registry (ACR).
```

**Review by mchammer01 — DISMISSED** [review]
@landongrindheim 👋🏻 - this LGTM ✨ 
Made a minor suggestion that I'm going to apply 🙂

**Review by mchammer01 — APPROVED** [review]
:shipit: ✨

**Pull request merged**